packages/commonwealth/client/styles/components/component_kit/new_designs/cw_radio_button.scss

```diff
@@ -27,14 +27,6 @@
     &:checked:hover {
       accent-color: $primary-600;
     }
-
-    &.disabled {
-      pointer-events: none;
-      box-shadow: inset 0 0 0 1px $neutral-200;
-      border-radius: 50%;
-      appearance: none;
-      background-color: $neutral-100;
-    }
```

Comment:
i think we shouldn't remove this as its used in https://github.com/hicommonwealth/commonwealth/blob/master/packages/commonwealth/client/scripts/views/components/poll_card.tsx

Reply:
1. This file is radio button from `component_kit/new_designs` and one from `poll_card` uses old radio button from `component_kit/cw_radio_button` so this is something different
2. I removed it as it did not work properly - when the value was selected but the state was disabled, there was no visual affordance that the radio is selected. Now it's look like this which is good 
<img width="173" alt="image" src="https://github.com/hicommonwealth/commonwealth/assets/14819225/85a61881-94ba-423e-b8c9-918b0bef5da5">